src/imp/libc/rand/syscalls.rs

```diff
@@ -5,6 +5,9 @@
 #[cfg(target_os = "linux")]
 use {super::super::c, super::super::conv::ret_ssize_t, crate::io, crate::rand::GetRandomFlags};
 
+// A duplicate weak symbol because we use this to detect the glibc version.
+weak! { pub(crate) fn libc_getrandom_weak(*mut c::c_void, c::size_t, c::c_uint) -> c::ssize_t }
```

Comment:
Can this macro invocation be placed inside the body of `is_atleast_glibc_2_25`, since we don't use it for actually calling `getrandom`?